PhoneFormat.js

```diff
@@ -192,15 +192,14 @@ Phoneformat.exampleMobileNumber = function (country) {
  * @param {String} phone
  */
 Phoneformat.cleanPhone = function (phone) {
-  phone = phone.replace(/[^\d\+]/g,'');
+  if (!phone || phone.length === 0) return '';
 
-  if (phone.substring(0, 1) == "+") {
-    phone = "+" + phone.replace(/[^\d]/g,'');
-  } else {
-    phone = phone.replace(/[^\d]/g,'');
-  }
+  var hasPlus = false;
+  if (phone.charAt(0) === "+") hasPlus = true;
+
+  phone = phone.replace(/[^\d]/g,'');
 
-  return phone;
+  return hasPlus ? '+' + phone : phone;
```

Comment:
@raix That would allow other pluses in the string to be entered. I think the formatting logic needs another refactoring pass in general sometime so we can separate out data/display and we can take a look at fixing this stuff up then.